library/src/webapp/js/portal/portal.login.password.js

```diff
@@ -28,7 +28,7 @@ class PasswordField {
   #handleCapslock(event) {
 
     const prevcapsLockActive = this._capsLockActive;
-    const capsMod = event.getModifierState("CapsLock");
+    const capsMod = event.getModifierState && event.getModifierState("CapsLock");
```

Comment:
I'm pretty sure you can use optional chaining here. But, why would getModifierState not be defined?

Reply:
I like it, I should have used 'optional chaining' here, my guess is that it's not defined because 'capslock' is not active at that moment. Thanks.